com.valvesoftware.SteamLink.yml

```diff
@@ -0,0 +1,208 @@
+app-id: com.valvesoftware.SteamLink
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: steamlink
+
+finish_args:
+  - --device=all
+  - --env=LD_LIBRARY_PATH=/app/lib
+  - "--env=SDL_GAMECONTROLLERCONFIG_FILE=/var/data/Valve Corporation/SteamLink/controller_map.txt"
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+
+tags:
+  - proprietary
+  - beta
+
+modules:
+  - name: qt5-qtbase
+    sources:
+      - type: archive
+        url: https://download.qt.io/archive/qt/5.14/5.14.1/submodules/qtbase-everywhere-src-5.14.1.tar.xz
+        mirror-urls:
+          - https://repo.steampowered.com/hidden/steamlink/d9d423a6e7bcf1055c0372fc029f14a6fe67dd62c67b83095cde68b60b762cf7/qtbase-everywhere-src-5.14.1.tar.xz
+        sha256: d9d423a6e7bcf1055c0372fc029f14a6fe67dd62c67b83095cde68b60b762cf7
+
+      - type: patch
+        path: patches/steamlink/qtbase.patch
+        strip-components: 2
+
+      # From org.kde.Sdk
+      - type: patch
+        path: patches/org.kde.Sdk/qtbase-avoid-hardcoding-kernel-version.patch
+      - type: patch
+        path: patches/org.kde.Sdk/qtbase-use-wayland-on-gnome.patch
+      # Temporary fix for https://github.com/flatpak/flatpak/issues/3397
+      - type: patch
+        path: patches/org.kde.Sdk/qtbase-revert-correct-handling-for-xdg-runtime-dir.patch
+      - type: patch
+        path: patches/org.kde.Sdk/qtbase-make-sure-to-correctly-construct-base-platform-theme.patch
+      - type: patch
+        path: patches/org.kde.Sdk/open-file-portal-writable.patch
+      - type: shell
+        commands:
+          - "mv configure configure.qt"
+      - type: script
+        commands:
+          - "processed=`sed -e 's/--/-/g ; s/=/ /g' <<< $@`"
+          - "./configure.qt $processed"
+        dest-filename: configure
+
+    # Cargo-culted from org.kde.Sdk
+    build-options:
+      arch:
+        x86_64:
+          config-opts:
+            - "-reduce-relocations"
+            - "-force-debug-info"
+        arm:
+          config-opts:
+            - "-no-reduce-relocations"
+            - "-optimize-size"
+            - "-opengl es2"
```

Comment:
```suggestion
```

`armhf` is no longer supported.

Reply:
(I know it's doesn't matter as per PR description but just so it doesn't give a wrong idea.)

Reply:
I'd slightly prefer to keep this in sync with the closest equivalent KDE runtime (which we're not using because we need to patch Qt anyway, and because it pulls in a lot of higher-level KDE stuff that we don't need). But if you feel strongly about this, I can drop it.